CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
-# dbt_microsoft_ads_source version.version
+# dbt_microsoft_ads_source v0.11.0
+[PR #39](https://github.com/fivetran/dbt_microsoft_ads_source/pull/39) includes the following changes:
+
+## Breaking Changes:
+- Updated `stg_microsoft_ads__ad_history` to include more name-related fields:
+  - `title`
+  - `title_part_1`
+  - `title_part_2`
+  - `title_part_3`
+  - `domain`
```

Comment:
I think it could be good to include the descriptions of these fields, or at least link to the `stg_microsoft_ads__ad_history` DAG page for more details

Reply:
added link to the dag page!